Samples/src/IntroToIQSharp/Notebook.ipynb

```diff
@@ -323,61 +352,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "Prepared |+⟩\n",
-      "Prepared |-⟩\n",
-      "Prepared |-⟩\n",
       "Prepared |+⟩\n",
       "Prepared |+⟩\n",
+      "Prepared |+⟩\n",
+      "Prepared |-⟩\n",
       "# wave function for qubits with ids (least to most significant): 0;1;2;3;4\n",
-      "0:\t0.176776695296637\t0\n",
-      "1:\t0.176776695296637\t0\n",
-      "2:\t-0.176776695296637\t0\n",
-      "3:\t-0.176776695296637\t0\n",
-      "4:\t-0.176776695296637\t0\n",
-      "5:\t-0.176776695296637\t0\n",
-      "6:\t0.176776695296637\t0\n",
-      "7:\t0.176776695296637\t0\n",
-      "8:\t0.176776695296637\t0\n",
-      "9:\t0.176776695296637\t0\n",
-      "10:\t-0.176776695296637\t0\n",
-      "11:\t-0.176776695296637\t0\n",
-      "12:\t-0.176776695296637\t0\n",
-      "13:\t-0.176776695296637\t0\n",
-      "14:\t0.176776695296637\t0\n",
-      "15:\t0.176776695296637\t0\n",
-      "16:\t0.176776695296637\t0\n",
-      "17:\t0.176776695296637\t0\n",
-      "18:\t-0.176776695296637\t0\n",
-      "19:\t-0.176776695296637\t0\n",
-      "20:\t-0.176776695296637\t0\n",
-      "21:\t-0.176776695296637\t0\n",
-      "22:\t0.176776695296637\t0\n",
-      "23:\t0.176776695296637\t0\n",
-      "24:\t0.176776695296637\t0\n",
-      "25:\t0.176776695296637\t0\n",
-      "26:\t-0.176776695296637\t0\n",
-      "27:\t-0.176776695296637\t0\n",
-      "28:\t-0.176776695296637\t0\n",
-      "29:\t-0.176776695296637\t0\n",
-      "30:\t0.176776695296637\t0\n",
-      "31:\t0.176776695296637\t0\n"
+      "∣ 0❭:\t 0.176777 +  0.000000 i\t == \t*                    [ 0.031250 ]     --- [  0.00000 rad ]\n",
```

Comment:
do we want all the output, or should all output just be [] ?

Reply:
Let me rerun to make sure I have all the output, thanks for that.